storages/backends/s3boto3.py

```diff
@@ -460,7 +460,10 @@ def _save(self, name, content):
 
     def delete(self, name):
         name = self._normalize_name(self._clean_name(name))
-        self.bucket.Object(name).delete()
+        try:
+            self.bucket.Object(name).delete()
+        except Exception as e:
+            print(e)
```

Comment:
Can you check for whatever specific error code there might be and just silence that and re-raise otherwise.

If you check the handling in `.exists()` and similar you will see what I mean.